homeassistant/components/persistent_notification/notify.py

```diff
@@ -0,0 +1,28 @@
+"""Persistent notification platform for notify integration."""
+from typing import Any, Optional
+
+from homeassistant.components.notify import BaseNotificationService
+from homeassistant.core import HomeAssistant
+
+from . import ATTR_MESSAGE, ATTR_TITLE, DOMAIN, SERVICE_CREATE
+
+
+async def async_get_service(
+    hass: HomeAssistant, config: dict, discovery_info: Optional[dict] = None
+) -> BaseNotificationService:
+    """Get the persistent notification service."""
+    return PersistentNotifyPlatform()
+
+
+class PersistentNotifyPlatform(BaseNotificationService):
+    """Implement the notification service for the persistent notification notify platform."""
+
+    async def async_send_message(self, message: str, **kwargs: Any) -> None:
+        """Create persistent notification from message."""
+        assert self.hass
```

Comment:
it would be odd for this to fail

Reply:
It would be odd, but mypy wants that or something like it, because the base class declares hass as `Optional[HomeAssistant]`, since it gets assigned outside of the constructor, so it is indeed unset for certain early method calls.